microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md

```diff
@@ -90,7 +25,7 @@ Uploads a package to update an existing draft submission of an add-on product.
 
 | Method | Request URI |
 |---|---|
-| `PUT` | `/products/{productID}/submissions/draft/package` |
+| `POST` | `/products/{productID}/submissions/draft/package` |
```

Comment:
Globally check in both files, whether the variable is named "productID" or "ProductID".

Reply:
It is productID